src/services/schema-sender.js

```diff
@@ -3,11 +3,23 @@ const agent = require('superagent-promise')(require('superagent'), P);
 const config = require('../config');
 const logger = require('../services/logger');
 
-function SchemaSender(serializedSchema, secret, oclifExit) {
+/**
+ * @class
+ * @param {string} serializedSchema
+ * @param {string} secret
+ * @param {(code: number) => void} oclifExit
+ * @param {string} authToken
+ */
+function SchemaSender(serializedSchema, secret, oclifExit, authToken) {
```

Comment:
```suggestion
function SchemaSender(serializedSchema, secret, authToken, oclifExit) {
```
would probably be a better signature as `secret` and `authToken` have a similar authentication/identification purpose and `oclifExit` seems here to handle errors.

Reply:
Fixed in #100